app/build.gradle

```diff
@@ -4,12 +4,12 @@ plugins {
 }
 
 android {
-    compileSdk 33
+    compileSdk 32
 
     defaultConfig {
         applicationId "com.portfolio.tasky"
         minSdk 30
-        targetSdk 33
+        targetSdk 32
```

Comment:
Why are you targetting 32 and not 33?

Reply:
The IDE caches are causing some problem when targeting to 33. I'll make sure they are reset and in the next PR I'll target it to 33.

Reply:
Sounds good!